test/indicators/loading-container.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { LoadingContainer } from '../../src/'
+
+
+test('LoadingContainer renders div with id "loading"', () => {
+  const wrapper = mount(
+    <LoadingContainer isLoading={true}>
+      <div id="loading">This text IS loading. </div>
+    </LoadingContainer>
+  )
+  expect(wrapper.find('div#loading').exists()).toBe(true)
+})
+
+test('LoadingContainer passes props', () => {
```

Comment:
This is testing the class-adding behavior- you should add another test for passing props to the div.

Reply:
Also, a test for when isLoading is false.